webapp/src/Controller/Jury/JudgehostController.php

```diff
@@ -117,6 +119,44 @@ public function indexAction(Request $request): Response
                 'value' => 'j' . $lastJobId['jobid'],
             ];
 
+            $now = Utils::now();
+            $timings = $this->em->createQueryBuilder()
+                ->from(JudgeTask::class, 'jt')
+                ->join(JudgingRun::class, 'jr', 'WITH', 'jr.judgetask = jt')
+                ->join('jr.judging', 'j')
+                ->join('jt.judgehost', 'jh')
+                ->select('jr.endtime, jr.startTime')
+                ->andWhere('jt.judgehost = :judgehost')
+                ->andWhere('jr.startTime IS NOT NULL')
+                ->andWhere('jr.endtime IS NOT NULL')
+                ->andWhere('jr.endtime >= :one_hour_ago')
+                ->setParameter('one_hour_ago', $now - 3600)
+                ->setParameter('judgehost', $judgehost)
+                ->getQuery()
+                ->getResult();
+            $loads = [0.0, 0.0, 0.0, 0.0];
+            $loadMinutes = [$now - 60*1, $now - 60*5, $now - 60*15, $now - 60*60];
+            foreach ($timings as $timing) {
+                for ($i = 0; $i < 4; $i++) {
+                    $start_time = $timing['startTime'];
+                    $end_time = $timing['endtime'];
+                    $start_time = max($start_time, $loadMinutes[$i]);
+                    if ($start_time < $end_time) {
+                        $loads[$i] += ($end_time - $start_time);
+                    }
+                }
+            }
+            // Normalize to [0,1] range.
+            $loads[0] = min(1.0, $loads[0] / (60.0 * 1.0));
+            $loads[1] = min(1.0, $loads[1] / (60.0 * 5.0));
+            $loads[2] = min(1.0, $loads[2] / (60.0 * 15.0));
+            $loads[3] = min(1.0, $loads[3] / (60.0 * 60.0));
```

Comment:
Maybe change above
```
          $loadMinutes = [60*1, 60*5, 60*15, 60*60];
```
and then reuse that data here again for deduplication?

Reply:
Done.